Makefile

```
@@ -251,6 +251,15 @@ endif
 clean:
 	$(RM) -r $(ROM) $(ELF) build
 
+clean_builtsrc:
+	$(RM) -r $(ROM) $(ELF) build/src build/data
+# Delete files directly inside build/ (but not in subfolders)
+	find build -maxdepth 1 -type f -exec $(RM) {} \;
+
+clean_builtassets:
```

```suggestion
builtassetsclean:
```
Wonder if this would be better to mirror `assetclean`? (same idea with `clean_builtsrc`)
Or perhaps `builtassets_clean` and `assets_clean`?